client/responses/responses.yaml

```diff
@@ -121,3 +126,52 @@ components:
         data:
           type: object
           $ref: "../models/models.yaml#/components/schemas/Rules"
+    LogWriteResponse:
+        type: object
+        required:
+          - status
+          - streams
+        properties:
+          error:
+            type: string
+          errorType:
+            type: string
+          warnings:
+            type: array
+            items:
+              type: string
```

Comment:
Can you provide a link/reference to validate these properties? I can't find anything in the protobuf or golang code.

Reply:
> Can you provide a link/reference to validate these properties? I can't find anything in the protobuf or golang code.

I carried it from metric properties